apps/authentication-service/cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5"],
+    "types": ["cypress", "node", "@testing-library/cypress"]
```

Comment:
You probably don't need @testing-library/cypress, because you won't be interacting with dom elements